package/thingino-webui/files/var/www/x/config-audio.cgi

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/haserl
+
```

Comment:
In haserl, every symbol below shebang goes directly to the output, including blank lines. Keep formatting like in other pages. Headers (in _common) go right next to the shebang.